PowerVCF.psm1

```diff
@@ -848,12 +844,7 @@ Function Request-VCFCertificateCSR {
         [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$json,
         [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$domainName
     )
-
-    if (!(Test-Path $json)) {
-        Throw "JSON File Not Found"
-    }
-    else {
-        $ConfigJson = (Get-Content -Raw $json) # Reads the requestCsrSpec json file contents into the $ConfigJson variable
+        validateJsonInput        
```

Comment:
Slipped in? 🙃

Reply:
d'oh! damn it, yes, slipped yet again from the other branch. sorry, let me restore it to how it was

Reply:
fixed